src/content/reference/react/use-server.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "'use server'"
+---
+
+<Wip>
+
+This section is incomplete.
+
+These directives are needed only if you're [using React Server Components](/learn/start-a-new-react-project#bleeding-edge-react-frameworks) or building a library compatible with them.
+
+</Wip>
+
+
+<Intro>
+
+`'use server'` marks server-side functions that can be called from client-side code.
+
+</Intro>
+
+<InlineToc />
+
+---
+
+## Reference {/*reference*/}
+
+### `'use server'` {/*use-server*/}
+
+Add `'use server';` at the very top of an async function to mark that the function can be executed by the client.
```

Comment:
```suggestion
Add `'use server';` at the very top of an async function to mark that the function can be executed by the server.
```
is it me or should this be executed by server?

Reply:
hm. I meant that it can be imported + triggered by the client